server/src/types/grades.ts

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+export interface Grade {
+  attainmentId: number,
+  points: number
+}
+
+export interface Student {
+  id?: number,
+  studentNumber: string,
+  grades: Array<Grade>
+}
+
+export interface AttainmentGrade {
```

Comment:
If this is intended to match the `UserAttainmentGrade` database model, it should be named `UserAttainmentGradeData` to be consistent with our other type naming. Granted, that is quite long so I can understand why you might have chosen not to name it that way, but I would personally still prefer it since it would immediately tell me that this is supposed to represent something from our database.

Reply:
updated to `UserAttainmentGradeData`